src/main/features/main.json

```diff
@@ -39,6 +39,22 @@
      {
         "id": "${project.groupId}:${project.artifactId}:${project.version}",
         "start-order": 25
-     }
+     },
+     {
+       "id": "org.apache.sling:org.apache.sling.jcr.api:2.4.0",
+         "start-order": 24
+     },
+     {
+       "id": "org.osgi:org.osgi.framework:1.10.0",
```

Comment:
You should not need to add it, the Felix framework should provide those packages already.